src/Analysis/Codelyzer.Analysis.VisualBasic/Handlers/PropertyBlockHandler.cs

```diff
@@ -14,19 +15,22 @@ public class PropertyBlockHandler : UstNodeHandler
             PropertyBlockSyntax syntaxNode)
             : base(context, syntaxNode, new PropertyBlock())
         {
-
-            foreach (var parameter in syntaxNode.PropertyStatement.ParameterList?.Parameters)
+            Model.Parameters = new List<Parameter>();
+            if (syntaxNode.PropertyStatement.ParameterList != null)
             {
-                var param = new Parameter
+                foreach (var parameter in syntaxNode.PropertyStatement.ParameterList.Parameters)
                 {
-                    Name = parameter.Identifier.ToString()
-                };
+                    var param = new Parameter
+                    {
+                        Name = parameter.Identifier.ToString()
+                    };
 
-                if (parameter.AsClause.Type != null)
-                    param.Type = parameter.AsClause.Type.ToString();
+                    if (parameter.AsClause.Type != null)
+                        param.Type = parameter.AsClause.Type.ToString();
 
-                param.SemanticType = SemanticHelper.GetSemanticType(parameter.AsClause.Type, SemanticModel, OriginalSemanticModel);
-                Model.Parameters.Add(param);
+                    param.SemanticType = SemanticHelper.GetSemanticType(parameter.AsClause.Type, SemanticModel, OriginalSemanticModel);
```

Comment:
if parameter.AsClause.Type is null? will this work? param.SemanticType = SemanticHelper.GetSemanticType(parameter.AsClause.Type, SemanticModel, OriginalSemanticModel);

Reply:
I don't think it would be valid syntax if the as clause has no type...but I'll wrap it in the if just in case.

Reply:
sure